test/metabase/query_analysis_test.clj

```diff
@@ -39,21 +49,27 @@
                                              {:joins [{:source-table (str "card__" (:id c2))
                                                        :alias        "Venues"
                                                        :condition    [:= $checkins.venue_id $venues.id]}]})}]
-      (mt/$ids
-       (is (= {:explicit #{%venues.name %venues.price}}
-              (#'query-analysis/query-field-ids (:dataset_query c1))))
-       (is (= {:explicit nil}
-              (#'query-analysis/query-field-ids (:dataset_query c2))))
-       (is (= {:explicit #{%venues.id %checkins.venue_id}}
-              (#'query-analysis/query-field-ids (:dataset_query c3)))))))
+      (is (= (mt/$ids
+              [{:table-id (mt/id :venues), :table "venues", :field-id %venues.name, :column "name", :explicit-reference true}
```

Comment:
this could be just `$$venues` under the `mt/$ids` macro

Reply:
TIL, thanks!

Reply:
Will clean this up in one of my later PRs, to save restacking.